### PR TITLE
fix(FEC-8554): update captions on resize event

### PR DESCRIPTION
**Pull request by odedhutzler**
### Description of the Changes

Added resize listener to the window, and updated the captions accordingly.
The `_resetTextCuesAndReposition` function acts as a `debounce` function, which will update the captions only after the last resize.

### CheckLists

- [x] changes have been done against master branch, and PR does not conflict
- [x] new unit / functional tests have been added (whenever applicable)
- [x] test are passing in local environment 
- [x] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

**Comment by odedhutzler** [comment]
@OrenMe no need to listen for `orientationchange`

**Comment by odedhutzler** [comment]
added unit test

File: test/src/player.spec.js [hunk 1]
```diff
@@ -2857,6 +2857,32 @@ describe('Player', function() {
...
+      config.sources = sourcesConfig.Mp4;
+      player = new Player(config);
+    });
+
+    afterEach(() => {
+      sandbox.restore();
+      player.destroy();
+    });
+
+    it('should call _resetTextCuesAndReposition function', done => {
```
Comment (OrenMe) [inline test/src/player.spec.js]: In general I think the correct title should be more declarative of what is happening, e.g. - 

> should re-calculate size and position of text cue

Comment (odedhutzler) [reply test/src/player.spec.js]: but it just calls that function and that's what the unit test checks.

there is another unit test that checks the _resetTextCuesAndReposition function itself.

**Pull request merged**